src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search__bar"
```

Comment:
Keep your attributes correctly formatted. If the HTML element has long attribute values or a number of attributes is more than 2, start each one on a new line with 2-space indentation related to the tag. The `class` attribute should start on a new line.